From c973b9b9be568ebbce618985bc5ee440babf8ab1 Mon Sep 17 00:00:00 2001
From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Thu, 27 Jun 2013 10:16:49 +0200
Subject: [PATCH 1/2] fix-glib-2.35-compatibility

g_thread_init() and g_thread_supported() are deprecated and don't
have to be used anymore.

---
 os.c |    9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

--- a/os.c
+++ b/os.c
@@ -4444,9 +4444,9 @@ fail:
 JNIEXPORT void JNICALL OS_NATIVE(_1g_1thread_1init)
 	(JNIEnv *env, jclass that, jintLong arg0)
 {
-	OS_NATIVE_ENTER(env, that, _1g_1thread_1init_FUNC);
+	/*OS_NATIVE_ENTER(env, that, _1g_1thread_1init_FUNC);
 	g_thread_init((GThreadFunctions *)arg0);
-	OS_NATIVE_EXIT(env, that, _1g_1thread_1init_FUNC);
+	OS_NATIVE_EXIT(env, that, _1g_1thread_1init_FUNC);*/
 }
 #endif
 
@@ -4454,11 +4454,12 @@ JNIEXPORT void JNICALL OS_NATIVE(_1g_1thread_1init)
 JNIEXPORT jboolean JNICALL OS_NATIVE(_1g_1thread_1supported)
 	(JNIEnv *env, jclass that)
 {
-	jboolean rc = 0;
+	/*jboolean rc = 0;
 	OS_NATIVE_ENTER(env, that, _1g_1thread_1supported_FUNC);
 	rc = (jboolean)g_thread_supported();
 	OS_NATIVE_EXIT(env, that, _1g_1thread_1supported_FUNC);
-	return rc;
+	return rc;*/
+	return 1;
 }
 #endif
 
-- 
1.7.10.4

